src/main/scala/no/ndla/articleapi/controller/InternController.scala

```diff
@@ -60,5 +64,41 @@ trait InternController {
     get("/embedurls/:external_subject_id") {
       ArticleContentInformation.getExternalEmbedResources(params("external_subject_id"))
     }
+
+    get("/reports/headerElementsInLists") {
+      contentType = "text/csv"
+      logger.info("Start searching for header elements in Lists in all articles")
+      val start = System.currentTimeMillis()
+      var errorMessages: List[HtmlFaultRapport] = immutable.List()
+      articleRepository.getAllIds.map(m => {
```

Comment:
Litt usikker på om all denne logikken bør ligge rett i kontrolleren. Mesteparten av denne kunne kanskje blitt flyttet til [ArticleContentInformation.scala](https://github.com/NDLANO/article-api/blob/master/src/main/scala/no/ndla/articleapi/service/ArticleContentInformation.scala), hvor vi allerede har funksjoner for liknende ting, for eksempel å liste ut alle forskjellige type tags som blir brukt i artikler.
Hva tror du?